tests/generation/SettingsTest.py

```diff
@@ -42,6 +48,33 @@ def test_write_info(self):
 
         self.assertEqual(result_comment, line)
 
+    def test_allow_complete_section_mode_with_ignore_globs(self):
+        project_dir = "/repo"
+        project_files = ['/repo/hello.html']
+        ignore_globs = ["/repo/style.css"]
+        used_languages = list(get_used_languages(project_files))
+        relevant_bears = filter_relevant_bears(
+            used_languages, self.printer, self.arg_parser, {})
+
+        res = generate_settings(
+            project_dir, project_files, ignore_globs, relevant_bears, {}, True)
+
+        bears_list = res["all.HTML"]["bears"].value.replace(" ", "").split(",")
+
+        files_list = res["all.HTML"]["files"].value.replace(" ", "").split(",")
+
+        print(res)
+        self.assertEqual(
+            ['HTMLLintBear', 'coalaBear', 'BootLintBear',
+             'LicenseCheckBear', 'SpaceConsistencyBear', 'KeywordBear',
+             'LineLengthBear', 'DuplicateFileBear'].sort(),
+            bears_list.sort())
+
+        ignore_file = str(self.path_leaf(
+            str(res["all"]["ignore"]).strip())).replace("\\", "/")
```

Comment:
You can use https://docs.python.org/2/library/os.path.html#os.path.normpath for this.